build.sbt

```diff
@@ -1,4 +1,10 @@
-libraryDependencies ++= Seq("com.typesafe" % "config" % "1.2.0")
+resolvers += "Sonatype-Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
```

Comment:
Why are we adding the sonatype snapshots repository as a resolver?  If we do want this, there's a variable for it in `sbt` instead of writing it out by hand.